app/controllers/course/condition/levels_controller.rb

```diff
@@ -1,12 +1,16 @@
 class Course::Condition::LevelsController < Course::ConditionsController
-  load_and_authorize_resource :level_condition, class: Course::Condition::Level.name, parent: false
+  load_resource :level_condition, class: Course::Condition::Level.name, parent: false
+  authorize_resource :level_condition, class: Course::Condition::Level.name, except: [:new, :create]
 
   def new
+    @level_condition.course = current_course
+    authorize!(:new, @level_condition)
```

Comment:
@lowjoel There is no course after resource was load. My solution is to manually authorize here after course was set.